lib/charms/mysql/v0/mysql.py

```diff
@@ -975,10 +975,9 @@ def get_cluster_primary_address(self, connect_instance_address: str = None) -> s
         Returns:
             The address of the cluster's primary
         """
-        logger.debug(f"Getting cluster primary member's address from {connect_instance_address}")
-
         if not connect_instance_address:
             connect_instance_address = self.instance_address
```

Comment:
do we need to patch the LIBPATCH version?

Reply:
i didn't because this does nothing functional. It will eventually be picked up by some update.
I've got annoyed by the messages:
`Getting cluster primary member's address from None`